variables.tf

```diff
@@ -242,6 +242,7 @@ variable "tfe" {
       value       = string
       category    = string
       description = string
+      sensitive   = bool
```

Comment:
Should we make this optional and and if no value is provided it will default to false?

Reply:
We should also see if can just merge the `workspace_vars` and `workspace_env_vars` as the only difference is if the category. But this should be done in a separate PR